src/IntrusivePtr.h

```diff
@@ -118,7 +118,7 @@ struct IntrusivePtr {
     }
 
     HALIDE_ALWAYS_INLINE
-    IntrusivePtr(IntrusivePtr<T> &&other) : ptr(other.ptr) {
+    IntrusivePtr(IntrusivePtr<T> &&other) noexcept: ptr(other.ptr) {
```

Comment:
micronit: space between noexcept and colon

Reply:
I'm looking around for more noexcept move constructors / assigns. I'll put in the space when I fix up those.